blog/what-is-slice.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: What is a Slice?
+description: What is a Slice? (Overlay Network & How it works in KubeSlice)
+slug: what-is-slice
+authors:
+  - name: Kunal Verma
+    url: https://github.com/verma-kunal
+hide_table_of_contents: false
+---
+
+[KubeSlice](https://kubeslice.io/documentation/open-source/0.3.0) is a new project in the cloud-native ecosystem that addresses the issues within a multi-cluster multi-tenant environment in Kubernetes.
+The project combines network, application, and deployment services in a framework to create multi-tenancy in a Kubernetes cluster and extends it to a multi-cluster architecture.
+
+It achieves this by creating a logical application boundary known as a **Slice** that allows pods and services to communicate efficiently across clusters, clouds, and data centers.  
+
+### What is a Slice?
+
+A **Slice** is very similar to the **Network Slice* that is predominant in 5G network architectures today. 
+[Network Slicing](https://en.wikipedia.org/wiki/5G_network_slicing) is a method of creating multiple unique **logical** and **virtualized** networks over a common, shared infrastructure. 
+We can also liken it to a virtual machine since it is similarly carved out of a large and common infrastructure to enable more efficient utilization & sharing of resources among multiple tenants.
+
+Each network slice is logically separated from the other, is an independent part of the network, and targets different services with different requirements at the same time. This enables to build and manage a network that has the capabilities to meet the emerging requirements from a wide range of users and enable multi-tenancy in a multi-cluster Kubernetes setup.
+
+### Concept of a Slice in KubeSlice
+
+In KubeSlice, a Slice is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) defined to implement multi-tenancy in a cluster. 
+[Multi-tenancy](https://kubernetes.io/docs/concepts/security/multi-tenancy/) is when a single instance of an application or a single Kubernetes cluster can serve multiple users, called tenants, at the same time without the problems of noisy/nosey neighbor, policy and configuration drift, and resource hogging creeping in.
+
+KubeSlice creates logical application **slice boundaries** in a cluster that allows pods and services to communicate efficiently within a cluster, across clusters on the cloud or datacenters. In order to communicate within a multi-cluster architecture, an overlay network (an application slice) following the RFC1918 protocol is created by KubeSlice that provides seamless connectivity between the pods of an application running in multiple clusters. Existing intra-cluster communication remains local to the cluster utilizing each pod’s CNI interface. 
+Kubeslice provides isolation of network traffic between clusters by creating an overlay network for inter-cluster communication.
```

Comment:
Can omit this line since you've already mentioned the overlay network following RFC1918 protocol above.
```suggestion
```